gp-includes/things/original.php

```diff
@@ -81,12 +81,28 @@ public function by_project_id( $project_id ) {
 	}
 
 	public function count_by_project_id( $project_id ) {
```

Comment:
To provide back-compat we can add a second parameter to switch between int (all counts) and stdClass (total,public,hidden).